pkg/api/v1alpha1/tinkerbellmachineconfig.go

```diff
@@ -125,3 +127,27 @@ func validateTinkerbellMachineConfig(config *TinkerbellMachineConfig) error {
 
 	return nil
 }
+
+func setTinkerbellMachineConfigDefaults(machineConfig *TinkerbellMachineConfig) {
```

Comment:
Are we doing these defaults just to be in sync with the cli or we determined that these are the defaults we want to move forward with?

Reply:
We're doing this to get in sync with the cli for the most part. We mainly error out on webhooks if we don't create a users field. We also have these today for other providers.